### PR TITLE
Set Spidermonkey 91 as default

### PR DESCRIPTION
**Pull request by big-r81**
Set Spidermonkey 91 as the new default version.

### CODE REVIEW

**Comment by janl** [comment]
let’s not remove support just yet. +1 for moving the default foward tho

File: configure.ps1 [hunk 1]
```diff
@@ -138,6 +138,7 @@ $CouchDBConfig = @"
 {node_name, "-name couchdb@localhost"}.
 {cluster_port, 5984}.
 {backend_port, 5986}.
+{prometheus_port, 17986}.
```
Comment (janl) [inline configure.ps1:141]: stray commit?

Comment (big-r81) [reply configure.ps1:141]: No, I only adapted this to the *nix configure-[version](https://github.com/apache/couchdb/blob/33ee2164e6e73429193bc353a2c7dfb9d27a7790/configure#L261)

Comment (janl) [reply configure.ps1:141]: ok, but this also has nothing to do with SM91? It’d be good to keep topical things in separate commits/PRs

Comment (big-r81) [reply configure.ps1]: ok, fixed. Should the doc update be separate too?

**Review by janl — APPROVED** [review]

**Pull request merged**